src/base/CellarInitializableV2_3.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import { Cellar, Registry, ERC20, PriceRouter } from "src/base/Cellar.sol";
+import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+
+contract CellarInitializableV2_3 is Cellar, Initializable {
+    /**
+     * @notice Constructor is only called for the implementation contract,
+     *         so it can be safely filled with mostly zero inputs.
+     */
+    constructor(
+        Registry _registry
+    )
+        Cellar(
+            _registry,
+            ERC20(address(0)),
+            "",
+            "",
+            abi.encode(new uint32[](0), new uint32[](0), new bytes[](0), new bytes[](0), 0, address(0), 0, 0)
+        )
+    {}
+
+    /**
+     * @notice Initialize function called by factory contract immediately after deployment.
+     * @param params abi encoded parameter containing
+     *               - Registry contract
+     *               - ERC20 cellar asset
+     *               - String name of cellar
+     *               - String symbol of cellar
+     *               - uint32 holding position
+     *               - bytes holding position config
+     *               - uint64 Strategist platform cut
+     *               - address Aave Pool contract address
+     * @dev Before calling `sendFees` in the FeesAndReserves contract INSURE that strategist payout is set!
+     */
+    function initialize(bytes calldata params) external initializer {
+        (
+            address _owner,
+            Registry _registry,
+            ERC20 _asset,
+            string memory _name,
+            string memory _symbol,
+            uint32 _holdingPosition,
+            bytes memory _holdingPositionConfig,
+            uint64 _strategistPlatformCut,
```

Comment:
So that amount is the percent of fees that go to the strategist. So if it is set to 0.8e18, then 80% of the fees go to the strategist, and 20% goes to the protocol. This was added as an input bc new cellars usually have different fee splits. so its a value we almost always change after deployment. But if we set it on initialization, then we dont need to worry about setting it later. Then I also added the aave pool address input because while all cellars deployed on ETH would use the same aave pool address, once we start deploying on mulitchain this is no longer the case. So we set it here on initialization. 

If you try to keep the strategist payout address as an input, then you run into stack too deep errors, and of the 3 things(The `_strategistPlatformCut`, `aavePool`, and `strategistPayoutAddress`), the `strategistPayoutAddress` is the easiest to change later because the strategist has the power to set it them selves, where as the `_strategistPlatformCut` arg must be changed by governance, and `aavePool` can ONLY be set during initialization, there is no function to change it.

Reply:
kk that all makes sense, thanks for the clarification!